src/app/map/map.component.ts

```diff
@@ -74,12 +80,27 @@ export class MapComponent implements AfterViewInit {
     this.updateStyles();
   }
 
-  constructor(private _ngZone: NgZone) {
+  constructor(private _ngZone: NgZone, private iterableDiffers: IterableDiffers) {
```

Comment:
nit: `iterableDiffers` isn't referenced outside the constructor, so doesn't need to be a private instance property.

Reply:
Done